docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md

```diff
@@ -54,7 +55,10 @@ Make sure that the following changes have been applied in transfer objects:
 | TRANSFER  | TYPE  | EVENT | PATH  |
 |-|-|-|-|
 | MerchantProductTableCriteria | class | Created | src/Generated/Shared/Transfer/MerchantProductTableCriteriaTransfer |
+| MerchantProductCriteria.idMerchant | property | Created | src/Generated/Shared/Transfer/MerchantProductCriteriaTransfer |
 | ProductAbstractCollection | class | Created | src/Generated/Shared/Transfer/ProductAbstractCollectionTransfer |
+| ProductTableCriteria | class | Created | src/Generated/Shared/Transfer/ProductTableCriteriaTransfer |
+| CriteriaRangeFilter | class | Created | src/Generated/Shared/Transfer/CriteriaRangeFilterTransfer |
```

Comment:
not needed, already specified in Marketplace core feature IG (exists not is this branch but in master)